apis/management.cattle.io/v3/zz_generated_deepcopy.go

```diff
@@ -4514,9 +4514,32 @@ func (in *LoadBalancerCapabilities) DeepCopy() *LoadBalancerCapabilities {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *LoadBalancerClassOpenstackOpts) DeepCopyInto(out *LoadBalancerClassOpenstackOpts) {
+	*out = *in
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new LoadBalancerClassOpenstackOpts.
+func (in *LoadBalancerClassOpenstackOpts) DeepCopy() *LoadBalancerClassOpenstackOpts {
+	if in == nil {
+		return nil
+	}
+	out := new(LoadBalancerClassOpenstackOpts)
+	in.DeepCopyInto(out)
+	return out
+}
+
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *LoadBalancerOpenstackOpts) DeepCopyInto(out *LoadBalancerOpenstackOpts) {
 	*out = *in
+	if in.LBClasses != nil {
```

Comment:
This is generated code and this looks like a hand edit. This will be overwritten.

Reply:
nope, this is a generated code